### PR TITLE
minor corrections

### PR DESCRIPTION
**Pull request by janhahne**
Just some minor corrections...

### CODE REVIEW

File: nestkernel/simulation_manager.cpp [hunk 1]
```diff
@@ -205,7 +205,7 @@ nest::SimulationManager::set_status( const DictionaryDatum& d )
   {
     if ( nprelim < 0 )
       LOG( M_ERROR,
-        "Scheduler::set_status",
+        "Network::set_status",
```
Comment (tammoippen) [inline nestkernel/simulation_manager.cpp]: Thanks for the catch! Also not quite correct: should be `"SimulationManager::set_status"`

Comment (janhahne) [inline nestkernel/simulation_manager.cpp]: Probably for all of them? Or just for those three?

Comment (tammoippen) [inline nestkernel/simulation_manager.cpp]: Yes: for all of them in this file.

**Comment by tammoippen** [comment]
Nice work!

**Pull request merged**